dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -475,7 +478,7 @@ object LagomPlugin extends AutoPlugin {
 
   private lazy val startKafkaServerTask = Def.task {
     val log = new SbtLoggerProxy(state.value.log)
-    val zooKeeperPort = lagomKafkaZookeperPort.value
+    val zooKeeperPort = lagomKafkaZookeeperPort.value.getOrElse(lagomKafkaZookeperPort.value)
```

Comment:
I'd rather not use `Option[Int]` either. 

I think @TimMoore's suggestion does work. I am not 100% sure of what's the behavior when users setup the port using `lagomKafkaZookeperPort` (deprecated name): in that case the default value (on `lagomKafkaZookeeperPort := lagomKafkaZookeperPort.value`  may still not see userland code).


** update**: I just tested and @timmoore's approach seems to work in all cases.